### PR TITLE
Json case

### PR DESCRIPTION
**Pull request by pdwetz**
Pull request for #53

### CODE REVIEW

File: src/DefaultJsonResponseNegotiator.cs [hunk 1]
```diff
@@ -19,7 +27,7 @@ public bool CanHandle(IList<MediaTypeHeaderValue> accept)
         public async Task Handle(HttpRequest req, HttpResponse res, object model, CancellationToken cancellationToken)
         {
             res.ContentType = "application/json; charset=utf-8";
-            await res.WriteAsync(JsonConvert.SerializeObject(model), cancellationToken);
+            await res.WriteAsync(JsonConvert.SerializeObject(model, new JsonSerializerSettings { ContractResolver = _contractResolver }), cancellationToken);
```
Comment (jchannon) [inline src/DefaultJsonResponseNegotiator.cs]: move the settings to the ctor and hardcode the camelcase resolver

File: test/ResponseNegotiatorTests.cs [hunk 2]
```diff
@@ -54,6 +56,33 @@ public ResponseNegotiatorTests()
             var response = await this.httpClient.GetAsync("/negotiate");
             Assert.Equal("application/json; charset=utf-8", response.Content.Headers.ContentType.ToString());
         }
+
+        [Fact]
+        public async Task Should_PascalCase_json()
```
Comment (jchannon) [inline test/ResponseNegotiatorTests.cs]: remove this test

File: src/DefaultJsonResponseNegotiator.cs (hunk 1 above)
Comment (pdwetz) [reply src/DefaultJsonResponseNegotiator.cs]: JsonSerializerSettings doesn't have an interface...

Comment (jchannon) [reply src/DefaultJsonResponseNegotiator.cs]: Sorry I meant the body if it
``` 
        public DefaultJsonResponseNegotiator()
         {
           this.jsonSettings = new JsonSerialzierSettings{ContractResolver = new CamelCasePropertyNamesContractResolver() };
         }
```

**Comment by jchannon** [comment]
👀

**Comment by jchannon** [comment]
Closed via #68

**Comment by jchannon** [comment]
Thanks @pdwetz your commits were used in the mentioned PR. 👍

**Pull request closed without merging**